client/src/browser-table.html

```diff
@@ -247,7 +247,7 @@ <h1 label>Browser support</h1>
       </div>
       <div class="row">
         <a spec href="https://w3c.github.io/webcomponents/spec/custom/" target="_blank" rel="noopener">
```

Comment:
This is now located at <code>https://html.spec.whatwg.org/multipage/custom-elements.html</code>

Reply:
@ExE-Boss Should we use https://www.w3.org/TR/html-imports/ instead of https://w3c.github.io/webcomponents/spec/imports/ for HTML Imports?

Reply:
Latter seems best.